lib/config_hash.rb

```diff
@@ -24,7 +24,7 @@ def initialize(verbose, trace)
         out_file.write(DEFAULT_CONFIG)
         out_file.close
         require require_file_name
-        
```

Comment:
Yep, good call. After everything looks good code-wise, we'll squash these commits into one,  remove the whitespace changes, and then submit that separately.